gefrickel

```diff
@@ -31,6 +36,11 @@ if [ -L lib ]; then
   pfx="usr/"
 fi
 
+if [ -n "$instsys_no_compression" ] ; then
+  echo "$instsys_no_compression" > .no_compression
+  echo "using no compression in squashfs\n"
```

Comment:
That's not what it actually does. In the extreme case the variable might be non-empty yet not match any of the options.